### PR TITLE
HPCC-19670 DOCS:Remove Graph Control

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

@JamesDeFabia please review
(for TheECLWatchMan.xml only need review new lines 425-426)
@miguelvazq  FYI; optionally review

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

Successful UNIT Testing results: http://10.240.32.243/view/Docs%207.0/job/DocDevII/

### CODE REVIEW

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml [hunk 1]
```diff
@@ -724,12 +724,10 @@
                   <entry><emphasis role="bold">ECL Watch Graph
                   Control</emphasis></entry>
 
-                  <entry>While the graph viewer control is built in to ECL
-                  IDE, ECL Watch requires this control to be installed so that
-                  it can display graphs properly. The currently installed
-                  version is identified and the version number of the new
-                  version shown. You should only adjust this setting if
-                  instructed to do so by customer support.</entry>
+                  <entry>The graph viewer control is built in to the ECL IDE.
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml]: This option is no longer in the IDE's Preferences > Other  tab. You should remove the entire row

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Comments inline

**Comment by AttilaVamos** [comment]
Master is fixed (related to xmlread error) and this PR is rescheduled.

**Comment by AttilaVamos** [comment]
Now it is upmerged and this PR rescheduled again.

**Comment by jakesmith** [comment]
tagging back to original reviewers : @JamesDeFabia @miguelvazq

**Review by JamesDeFabia — APPROVED** [review]
Looks good now

**Comment by richardkchapman** [comment]
@GordonSmith Please review

**Review by GordonSmith — APPROVED** [review]
Looks ok - but appears to have a bunch of unrelated changes?

**Pull request merged**